recipe/meta.yaml

```diff
@@ -7,14 +7,14 @@ package:
 
 source:
   - url: http://download.osgeo.org/proj/proj-{{ version }}.tar.gz
-    sha256: 6b1379a53317d9b5b8c723c1dc7bf2e3a8eb22ceb46b8807a1ce48ef65685bb3
+    sha256: ef919499ffbc62a4aae2659a55e2b25ff09cccbbe230656ba71c6224056c7e60
   - url: http://download.osgeo.org/proj/proj-datumgrid-{{ datumgrid_ver }}.zip
     sha256: ffcad4453fac3b4a13ca678ef343e688496e18019f35709ef03c5f200e9fb85a
     folder: nad
 
 
 build:
-  number: 1002
+  number: 0
```

Comment:
@CJ-Wright and @mariusvniekerk what is the policy on updates for those feedstocks that are already using `gcc7`? Should this be bumped to 1003?

Reply:
Doh! Of course... Thanks!!

Reply:
@CJ-Wright we need a plan to avoid bot updates resetting this to `0` before the migration is over. Not sure what is the best way to do that though.

Reply:
I think there is an open issue for this. We'd need to make the `Version` migrator a bit smarter about tracking the version number, although we may have some logic for blas numbers.

Reply:
I cannot wait for this extra complexity to go away once we finish the compiler migration and get our act together on the variants.